cypress/e2e/19-execution.cy.ts

```diff
@@ -1,3 +1,12 @@
+import { clickGetBackToCanvas, getOutputTableRow } from '../composables/ndv';
+import {
+	clickExecuteWorkflowButton,
+	getExecuteWorkflowButton,
+	getNdv,
```

Comment:
small nitpick/optional - we can import this as `getNdvContainer` directly from `ndv` file.
Might be simpler than duplicating `getNdv` and `getContainer` that do the same thing.